src/crystal/system/event_loop.cr

```diff
@@ -0,0 +1,32 @@
+module Crystal::EventLoop
```

Comment:
I figured the `Crystal::System` namespace is only really useful when there are both a system-specific type and a general type in `Crystal`. Otherwise it only adds clutter.
There is however the possibility of name clashes with the compiler which also uses the `Crystal` namespace. But that's a general issue unless we put *everything* in `Crystal::System`.

Reply:
I put everything which has a platform-specific implementation - and is using the new `Crystal::System` structure for it in `Crystal::System`.

I'm fine with not renaming, but I personally would have done it, since the files are in `src/crystal/system`.

Reply:
I see it that way that the things in `Crystal::System` are not used anywhere directly except as an implementation detail of their specific counterpart in the public API. `EventLoop`, `Thread` etc. however are general APIs for Crystal's runtime. They only have system-specific implementations, but they're directly used in the runtime. `Crystal::EventLoop` for example is referenced 10 times in the stdlib (without specs), `Crystal::Thread` 17 times.